romanisim/tests/test_bandpass.py

```diff
@@ -50,24 +58,179 @@ def test_compute_abflux(filter):
 
     # Create dirac-delta-like distribution for filter response
     wavedist = np.linspace(wavel.value - 0.001, wavel.value + 0.001, 1000)
-    thru = norm.pdf(wavedist, wavel.value, 0.0001)
+    flux = norm.pdf(wavedist, wavel.value, 0.0001)
 
     # Analytical flux
-    theo_flux = (abfv * area / (constants.h.to(u.erg * u.s) * wavel)).to(1 / (u.s * u.micron))
+    an_flux = (abfv * area / (constants.h.to(u.erg * u.s) * wavel)).to(1 / (u.s * u.micron))
 
     # Table for filter data storage
     data_table = Table()
     data_table['Wave'] = wavedist
-    data_table[filter] = thru
+    data_table[filter] = flux
 
     # Computed flux
     gauss_flux = bandpass.compute_abflux(data_table)
 
     # Comparing both fluxes as magnitudes
-    assert np.isclose(np.log10(theo_flux.value), np.log10(gauss_flux[filter]), atol=1.0e-6)
+    assert np.isclose(np.log10(an_flux.value), np.log10(gauss_flux[filter]), rtol=1.0e-6)
```

Comment:
In this case we were changing the throughputs rather than the fluxes; I now understand where the confusion about throughput and flux is coming from.

Here we simulated a filter with some delta function throughput and integrated a flat spectrum (abfv) across it.  So thru & theo_flux are appropriate; I don't think these changes are needed.

Reply:
Undone.